core/src/main/java/org/kiwix/kiwixmobile/core/help/HelpFragment.kt

```diff
@@ -88,16 +89,21 @@ abstract class HelpFragment : BaseFragment() {
   }
 
   private fun sendFeedback() {
-    val intent = Intent(Intent.ACTION_SENDTO)
-    intent.data = (
-      "mailto:${Uri.encode(CONTACT_EMAIL_ADDRESS)}" +
-        "?subject=" +
-        Uri.encode(
-          "Feedback in " +
-            getCurrentLocale(requireActivity()).displayLanguage
-        )
-      ).toUri()
-    startActivity(Intent.createChooser(intent, "Send Feedback via Email"))
+    val emailIntent = Intent(Intent.ACTION_SENDTO).apply {
+      data = (
+        "mailto:${Uri.encode(CONTACT_EMAIL_ADDRESS)}" +
+          "?subject=" +
+          Uri.encode("Feedback in ${getCurrentLocale(requireActivity()).displayLanguage}")
+        ).toUri()
+    }
+    val packageManager = requireActivity().packageManager
+    val activities = packageManager.queryIntentActivities(emailIntent, 0)
```

Comment:
@criticalAY , ```queryIntentActivities(intent,int)``` is deprecated in android 13. Please use ```queryIntentActivities(intent,ResolveInfoFlags)``` to support android 13.

like 

```
 val activities = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
      packageManager.queryIntentActivities(
        emailIntent,
        PackageManager.ResolveInfoFlags.of(MATCH_DEFAULT_ONLY.toLong())
      )
    } else {
      packageManager.queryIntentActivities(
        emailIntent,
        MATCH_DEFAULT_ONLY
      )
    }
```